src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/KlageBlankettSaksbehandlingMetadata.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.Fagsystem
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.dokarkiv.Dokumenttype
+import org.springframework.stereotype.Component
+
+abstract class KlageBlankettSaksbehandling(
+    override val fagsakSystem: Fagsystem,
+    final override val tema: Tema,
+    final override val behandlingstema: Behandlingstema,
+    override val dokumenttype: Dokumenttype,
+) : Dokumentmetadata {
+
+    override val journalpostType: JournalpostType = JournalpostType.NOTAT
+    override val kanal: String? = null
+    override val tittel: String? = null
+    override val brevkode: String = "KLAGE_BLANKETT_SAKSBEHANDLING_${behandlingstema.name.uppercase()}"
```

Comment:
Usikker på behandlingstema her:
Skal vi journalføre på "ENF-Overgangsstønad" [tema+behandlingstema] eller "ENF-klage" [tema+behandlingstype] ?

Reply:
Bra spørsmål! Tenker det kan være fint å ha med behandlingstema her, då den indirekt sier hvilken behandling den er. Vet ikke om vi kommer bruke eks BarnetrygdEØS? Eller om vi kun kommer bruke "våre stønadstyper"
https://nav-it.slack.com/archives/G012YEK9YQ1/p1644411964839829?thread_ts=1644411613.773309&cid=G012YEK9YQ1 du skrev her at det kan være fint å ha med stønadstypen 😬